index.html

```diff
@@ -15,5 +15,73 @@
     <!-- Remember: Use semantic HTML tags like <header>, <main>, <nav>, <footer>, <section> etc -->
     <!-- All the images you need are in the 'img' folder -->
 
+    <header class="header-container">
+        <div class="logo">
+            <img src="/img/karma-logo.svg" 
+            alt="karmalogo" />
+        </div>
+           
+        <div class="nav-bar">
+                <label for="toggle">&#9776;</label>
+                <input type="checkbox" id="toggle"/>
+                <div class="menu">
+                
+                        <a href="#"><span>Meet Karma</span></a>
+                        <a href="#">How it Works</a>
+                        <a href="#">Store</a>
+                        <a href="#">Blog</a>
+                        <a href="#">Help</a>
+                        <a href="#">Login</a>
+                    
+                </div>
```

Comment:
Great work with making the navbar responsive! My only comment here would be to wrap it in a `<nav>` tag to make it more responsive

Reply:
It was <nav>, but a friend helped me to fix my hamburger menu, then she replaced some of my tags. I'll do it, thanks!